pkg/kubectl/scale.go

```diff
@@ -216,6 +227,9 @@ func (scaler *JobScaler) ScaleSimple(namespace, name string, preconditions *Scal
 		}
 	}
 	parallelism := int(newSize)
+	if job.Spec.Parallelism != nil && *job.Spec.Parallelism == parallelism {
```

Comment:
I cringe in the face of accessing a pointer w/o checking for nil first. I am going to leave this around since it doesn't harm having safer code:)